pom.xml

```diff
@@ -249,6 +249,13 @@
             <artifactId>google-api-services-sheets</artifactId>
             <version>v4-rev612-1.25.0</version>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
```

Comment:
This dependency creates a problem while running tests using Junit. Did you detailed unit testing?

Reply:
I have tested it and it is working fine. 
https://github.com/onlyforcoding24/FrameSwitch.git
Run this tag @buyPillowSuccessfully from above project which contains normal assertion and api assertion. Both are working fine.

Reply:
Noted.